### ISSUE TITLE
How does embed file work?

### ISSUE THREAD

**Issue by oh-facts** [mariakeating/idleblock#1]
If I understand correctly, this is to avoid having to read the files using io? I have a million other questions I have never used dbus.

**Comment by mariakeating** [comment]
Yep! incbin embeds the file at compile time so it's just a block of read-only memory. The rest of the instructions are just setting to the correct location and recording the size of the memory block.
